boot/zephyr/main.c

```diff
@@ -536,6 +536,9 @@ int main(void)
 
     BOOT_LOG_INF("Bootloader chainload address offset: 0x%x",
                  rsp.br_image_off);
+    BOOT_LOG_INF("Image version: v%d.%d.%d", rsp.br_hdr->ih_ver.iv_major,
```

Comment:
This should be configurable, because we start to take extra space with things that will be rarely viewed by anyone.

Reply:
I would imagine those concerned deeply about space will have logging off anyway. I'd hate to see yet another ifdef in the code just for this.